src/routes/api/api-resync-post.ts

```diff
@@ -23,8 +38,20 @@ export const ApiResyncPost = async (req: Request, res: Response): Promise<void>
 	// restrict sync to a subset of tasks
 	const targetTasks = req.body.targetTasks as TaskType[];
 
-	if (!statusTypes && !installationIds && !limit && !inactiveForSeconds) {
-		res.status(400).send("please provide at least one of the filter parameters!");
+	if (!statusTypes && !gitHubInstallationIds && !limit && !inactiveForSeconds) {
+		res.status(400).send("Please provide at least one of the filter parameters!");
+		return;
+	}
+
+	if (!gitHubInstallationIds.length) {
```

Comment:
/nit
for `!gitHubInstallationIds.length`, say [1,2,3] are provided as `gitHubInstallationIds`, but only return [2,3], do we want to make it 400 or still allowing it?

Reply:
I'd thought about that - it's primarily used by us to resync folks and I figured it would be a pretty bad experience if we had a CSV with a large number of gh installationIds but accidentally entered a couple incorrectly. It would mean it would fail on one, we'd update it and run again, then it would fail on the second one etc. 

We could potentially try and return all incorrect ids so they could be removed in one go. Having said that though, while we don't need to worry about ppl passing in bogus data and getting a successful response (it would fail when we call Subscription.getAllFiltered), we should probably return a 400 if no subscriptions are found.